src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <>
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    </>
+    <p className="Person__partner">
+      {person.isMarried !== false ? (
+        `${person.partnerName} is my ${person.sex === 'm' ? (
+          `wife`
+        ) : (
+          `husband`
+        )}`
+      ) : (
+        `I am not married`
+      )}
```

Comment:
make nested ternary operators is a bad practice
you can move this logic to the variables